variables.tf

```diff
@@ -12,13 +12,17 @@ variable "records" {
   description = "List of names to create"
   type = list(object({
     name     = string
-    value    = string
+    value    = optional(string)
     type     = string
     ttl      = number
-    proxied  = bool
-    priority = number
+    proxied  = optional(bool)
+    priority = optional(number)
```

Comment:
You have to address it also where you use this field. In case `priority` is absent.

Reply:
> You have to address it also where you use this field. In case `priority` is absent.

Can you please elaborate or provide a failing test case?

Reply:
Should you use `lookup` function here?

https://github.com/Flaconi/terraform-cloudflare-records/blob/a778d4ce612984678377dd090100ab405ded4c6c/main.tf#L18

Reply:
solved